saleor/graphql/shop/schema.py

```diff
@@ -40,9 +50,16 @@ class ShopQueries(graphene.ObjectType):
     def resolve_shop(self, _info):
         return Shop()
 
-    @load_site_callback
-    def resolve_order_settings(self, _info, site):
-        return site.settings
+    def resolve_order_settings(self, _info):
+        channel = channel_models.Channel.objects.order_by("slug").first()
```

Comment:
Also not sure that we should return this value from the first channel ordered by slug. 
Maybe better to fetch the first active channel ordered by pk?
At lest we should filter out deactivated channels.

Reply:
We want to have a slug here as we add this info in the field description. Finding first channel for user sorted by slug is easier than by PK.